WNPRC_Purchasing/src/client/components/LineItemRow.tsx

```diff
@@ -47,52 +52,78 @@ export const LineItemRow: FC<LineItemProps> = memo(props => {
 
     return (
         <>
-            <Row className="line-item-row" key={'line-item-row-' + rowIndex}>
-                <Col xs={4}>
-                    <DescriptionInput
-                        value={model.item}
-                        hasError={model.errors?.find(field => field.fieldName === 'item')}
-                        onChange={onValueChange}
-                    />
-                </Col>
-                <Col xs={1}>
-                    <UnitInput
-                        value={model.itemUnit}
-                        hasError={model.errors?.find(field => field.fieldName === 'itemUnit')}
-                        onChange={onValueChange}
-                    />
-                </Col>
-                <Col xs={1}>
-                    <UnitCostInput
-                        value={model.unitCost}
-                        hasError={model.errors?.find(field => field.fieldName === 'unitCost')}
-                        onChange={onValueChange}
-                    />
-                </Col>
-                <Col xs={1}>
-                    <UnitQuantityInput
-                        value={model.quantity}
-                        hasError={model.errors?.find(field => field.fieldName === 'quantity')}
-                        onChange={onValueChange}
-                    />
-                </Col>
-                <Col xs={1}>
-                    <SubtotalInput unitCost={model.unitCost} quantity={model.quantity} />
-                </Col>
-                <Col xs={2}>
-                    <ControlledSubstance value={model.controlledSubstance} onChange={onValueChange} />
-                </Col>
-                <Col xs={2}>
-                    <span
-                        id={'delete-line-item-row-' + rowIndex}
-                        title="Delete item"
-                        className="delete-item-icon"
-                        onClick={onDeleteRow}
-                    >
-                        <FontAwesomeIcon className="fa-faTimesCircle" icon={faTimesCircle} />
-                    </span>
-                </Col>
-            </Row>
+            {
+                <Row className="line-item-row" key={'line-item-row-' + rowIndex}>
+                    <Col xs={4}>
+                        <DescriptionInput
+                            value={model.item}
+                            hasError={model.errors?.find(field => field.fieldName === 'item')}
+                            onChange={onValueChange}
+                            isReadOnly={!isAdmin && canUpdate}
+                        />
+                    </Col>
+                    <Col xs={1}>
```

Comment:
xs={1} is pretty small for an input.  Did you verify it holds up on smaller width screens?  Same with the inputs below.

Reply:
It is as expected